examples/ffi/symbolizer.cpp

```diff
@@ -0,0 +1,46 @@
+#include <iostream>
+#include <vector>
+
+#include <unistd.h>
+#include <cassert>
+
+#include "datadog/symbolizer.h"
+
+#define _THIS_IP_                                                              \
+  ({                                                                           \
+    __label__ __here;                                                          \
+  __here:                                                                      \
+    (unsigned long)&&__here;                                                   \
+  })
+
+// Virtual only works as privileged user
```

Comment:
@danielsn I am puzzled as to why this would not work without specific privileges, even within the same process. Have you investigated this API by any chance?